src/Command/Ide/Wizard/IdeWizardCommandBase.php

```diff
@@ -71,9 +71,9 @@ protected function getSshKeyLabel() {
   /**
    * @throws \Acquia\Cli\Exception\AcquiaCliException
    */
-  protected function deleteThisSshKeyFromCloud(): void {
+  protected function deleteThisSshKeyFromCloud($output): void {
```

Comment:
This fixes a latent bug (missing parameter) that prevents you from deleting SSH keys on cloud. We need to make sure it gets committed regardless of what happens to this PR.